uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -379,13 +379,24 @@ define([
         }
 
         if (missingRequiredFields.length) {
-          var notification = CRM.alert(missingRequiredFields.join(', '),
-            missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+          var notification;
+          var notificationTitle = missingRequiredFields.length === 1 ? 'Required field' : 'Required fields';
+          var missingFields = missingRequiredFields.join(', ');
+
+          if ($scope.isRole('manager')) {
+            $rootScope.$broadcast('ssp:notify', {
+              type: 'error',
+              title: notificationTitle,
+              body: missingFields
+            });
+          } else {
+            notification = CRM.alert(missingFields, notificationTitle);
```

Comment:
@deb1990 I've already proposed with the approach you mentioned in technical technical description of the ticket. And it needs few changes in backend code. Having discussion with @AkA84, we decided to find alternative solution and came to use toastr.